root/taglookup/Results.js

```diff
@@ -15,17 +15,18 @@ import TagLookupForm from './Form';
 import TagLookupNagSection from './Nag';
 import type {TagLookupResultsPropsT} from './types';
 
-const TagLookupResults = <T>(
-  props: TagLookupResultsPropsT<T>,
-): React.Element<typeof Layout> => (
-  <Layout $c={props.$c} fullWidth title={l('Tag Lookup Results')}>
-    <div className="content">
-      <h1>{l('Tag Lookup Results')}</h1>
-      {props.nag ? <TagLookupNagSection /> : null}
-      {props.children}
-      <TagLookupForm form={props.form} />
-    </div>
-  </Layout>
-);
+const TagLookupResults = <T>
+  (
+    props: TagLookupResultsPropsT<T>,
+  ): React.Element<typeof Layout> => (
+    <Layout $c={props.$c} fullWidth title={l('Tag Lookup Results')}>
```

Comment:
This is the only way I found to make both indent and jsx-indent happy at the same time.